packages/common/store/index.js

```diff
@@ -1,17 +1,20 @@
 import {
   createStore, combineReducers, applyMiddleware, compose,
 } from 'redux';
-
+import storage from 'redux-persist/lib/storage';
+import { persistStore } from 'redux-persist';
 import actionTypes from './actions/actionTypes';
 import * as reducers from './reducers';
 import middleWares from './middlewares';
 
-const App = combineReducers(reducers);
-const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+export * from './selectors';
 
-const store = createStore(App, composeEnhancers(applyMiddleware(...middleWares)));
+const rootReducer = combineReducers(reducers);
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
+const store = createStore(rootReducer, composeEnhancers(applyMiddleware(...middleWares)));
 store.dispatch({ type: actionTypes.storeCreated });
+const persistedStore = persistStore(store);
```

Comment:
These changes are not relevant. Please open a ticket, explain your goal and submit changes via the corresponding PR.

Reply:
@reyraa I already explain it to the team and its my pleasure to briefly explain to you, I believe this change is part of the acceptance criteria!

Reply:
no need. If other agreed, it's fine.